.github/workflows/update-dotnet-install-script.yml

```
@@ -0,0 +1,48 @@
+name: "Update dotnet-install script"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 0 * * 0'  # Runs every Sunday at midnight UTC (adjust as needed)
+
+jobs:
+  fetch-latest-dotnet-install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run fetch-latest-dotnet-install.sh
+        run: src/dotnet/scripts/fetch-latest-dotnet-install.sh
+
+      - name: Create a PR for dotnet-install.sh
+        id: push_image_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.PAT }}
```

Unfortunately, at org level we have the setting for "Allow GitHub to create and merge PRs" option disabled.